packages/components/src/components/text-field/text-field.tsx

```diff
@@ -95,7 +95,8 @@ export class TextField {
 
   /** (optional) Injected CSS styles */
   @Prop() styles?: string;
-
+  /** (experimental)) Makes type `input` behave as a controlled component in React */  
```

Comment:
there's an extra ) in `(experimental))` 🤓

Reply:
> Would it work with the snake_case? 🤔

seems like it doesn't work, the linter complains. experimentalControlled/experimental-controlled works

Reply:
I'm not in love with `experimental-controlled` but I think I prefer it over just `controlled` (only because it's a React-only thing)